services/datamanager/builtin/builtin.go

```diff
@@ -103,9 +103,14 @@ type builtIn struct {
 	cloudConnSvc        cloud.ConnectionService
 	cloudConn           rpc.ClientConn
 	syncTicker          *clk.Ticker
+
+	corruptedFilesDir string
 }
 
-var viamCaptureDotDir = filepath.Join(os.Getenv("HOME"), ".viam", "capture")
+var (
+	viamCaptureDotDir   = filepath.Join(os.Getenv("HOME"), ".viam", "capture")
+	viamCorruptedDotDir = filepath.Join(os.Getenv("HOME"), ".viam", "corrupted_capture")
```

Comment:
We expose the capture directory in the config. I'm wondering if this needs to be, too. Alternatively, if it were made a subdirectory of the capture directory I don't think that would be necessary. That would require updating the capture directory trawling code to ignore files in that subdirectory, but I (think) that would be a simple change

Reply:
I could see both as being a viable option, tbh. On one hand, I think nesting subdirectory would allow us to expose one less thing in the config. On the other hand, I think having a kind of mirrored structure to the corrupted files also makes sense. 

As an example, having it as a subdirectory would look like:
`/Users/tahiyasalam/.viam/capture/corrupted_capture/rdk:component:camera/camera/ReadImage/bad_data.prog`, while simultaneously the directory `/Users/tahiyasalam/.viam/capture/rdk:component:camera/camera/ReadImage/` would also exist.

I don't find this to be a huge deal so happy to go either way, but I personally think having the two distinct is slightly easier visually and reason about. What do you think?

Reply:
We could also just append "_corrupted" to whatever directory is supplied, so we would still have two distinct directories and not expose this in the config.

Reply:
I think I'm still partial to a subdirectory. I think it's almost sort of a permissions assumption/leap to do the parallel `_corrupted` directory, in the sense that a user specifying that things can be written to a directory wouldn't necessarily want rdk to be able to create directories at the same level. I think it would get a little complicated if a user ever provides a custom path to be written to where rdk has specific permissions (e.g. write permissions in the directory, but not write permissions in the parent directory, which I think would be necessary to allow the creation of the _corrupted directory)

Reply:
Yup yup, that totally makes sense. I will refactor to a subdirectory based on this.